packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.node.spec.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * @jest-environment node
+ */
+
+import * as React from "react";
+import * as ReactDOMServer from "react-dom/server";
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure, shallow } from "enzyme";
+import HorizontalOverflow, {
+    IHorizontalOverflowClassNameContract,
+    IHorizontalOverflowHandledProps,
+} from "./";
+
+/*
+ * Configure Enzyme
+ */
```

Comment:
Do we use `.node.spec` anywhere else? I'm guessing you're trying to highlight that these run in a "node" environment?

I think we should keep all our tests together for now until we can discuss organization as part of our e2e testing work.